assets/css/index.css

```diff
@@ -1,75 +1,126 @@
-@import '/assets/css/reset.css';
-@import '/assets/css/nav.css';
-@import '/assets/css/footer.css';
-
+@import "/assets/css/reset.css";
+@import "/assets/css/nav.css";
+@import "/assets/css/footer.css";
 
 /* body & image section */
 
-.comp-lib-sub-body{
-    display: flex;
-    flex-direction: column;
-    justify-content: space-evenly;
-    flex-wrap: wrap;
-    padding-left: 3rem;
+.comp-lib-nav {
+  height: 100vh;
+  width: 100%;
+  background-color: var(--off-white);
+  transition: 0.4s ease-in-out;
 }
 
-.comp-lib-body{
-    display: flex;
-    align-items: center;
-    margin-left: 1rem;
-    margin-right: 1rem;
-    width: 100%;  
+.nav-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
 
-.comp-lib-body-ele{
-    display: flex;
-    flex-direction: column;
+#dark-mode {
+  /* margin-right: 2rem; */
+  cursor: pointer;
+}
+
+.comp-lib-nav-wrapper {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.comp-lib-sub-body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
```

Comment:
Suggestion: you could make utility classes to reduce the repetition of code. 

demo:

/*  utility class  Just an example*/
.flex {
  display: flex:
  justify-content: center;
  align-items: center;
}

/*  container specific flex property  */
.someclass .flex {
  justify-content: space-between;
 }
 
 This way of using the utility class will save us from writing the same properties again and again.